src/metabase/driver/presto.clj

```diff
@@ -350,7 +348,6 @@
           :current-datetime-fn       (constantly :%now)
           :date                      (u/drop-first-arg date)
           :excluded-schemas          (constantly #{"information_schema"})
-          :field-percent-urls        (u/drop-first-arg field-percent-urls)
```

Comment:
... this is just crazy. So it seems like `field-percent-urls` is being taking out of the `IDriver` interface (which is fine) but the corresponding implementations are just left there, doing nothing. Dead code *all over the place* like that makes things extremely confusing. Plus I've had to clean up so much of this code I'm worried I may have inadvertently broke something